CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -359,6 +360,8 @@ - (void)initWithChannelId {
     NSUserDefaults* userDefaults = [NSUserDefaults standardUserDefaults];
     subscriptionId = [userDefaults stringForKey:CLEVERPUSH_SUBSCRIPTION_ID_KEY];
     deviceToken = [userDefaults stringForKey:CLEVERPUSH_DEVICE_TOKEN_KEY];
+    deviceID = [[[UIDevice currentDevice] identifierForVendor] UUIDString];
```

Comment:
Remove here. Only call this below:
https://github.com/cleverpush/cleverpush-ios-sdk/pull/84/files#diff-acb697613e56d60a503bc85a50eb9229383c0a24a923945f80f600bc5785aefcR1214